### ISSUE TITLE
npm init just fails

### ISSUE THREAD

**Issue by moltar** [microsoft/just#198]
macOS Mojave 10.14.5 (18F132)

```
❯ node -v
v10.15.3
```

```
❯ npm init just
(node:75833) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'startsWith' of undefined
    at expandHome (/Users/foo/.npm/_npx/75736/lib/node_modules/create-just/lib/getEnvInfo.js:29:14)
    at Object.<anonymous> (/Users/foo/.npm/_npx/75736/lib/node_modules/create-just/lib/getEnvInfo.js:24:69)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/foo/.npm/_npx/75736/lib/node_modules/create-just/lib/getEnvInfo.js:4:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:75833) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:75833) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by kenotron** [comment]
Good catch on systems without yarn installed!

**Comment by kenotron** [comment]
Fixed in #201
